src/quick-lane/quick-lane.service.ts

```diff
@@ -18,13 +23,19 @@ import {
 export interface QuickLaneUrl {
 	id: string;
 	title: string;
+	content?: AssignmentContent;
 	content_id?: string;
 	content_label?: AssignmentContentLabel;
+	owner?: QuickLaneUrlOwner;
 	owner_profile_id?: string;
 	created_at?: string;
 	updated_at?: string;
 }
 
+export interface QuickLaneUrlOwner extends Pick<UserProfile, 'id' | 'avatar'> {
+	usersByuserId: Pick<UserSchema, 'full_name'>;
```

Comment:
Muggenziften, maar de `user` heeft geen hoofdletter

Reply:
https://hasura-graphql-qas-avo.private.cloud.meemoo.be/console/data/schema/users/tables/profiles/browse, laatste kolom is ook met kleine `u` 😬